docs/_static/custom.css

```diff
@@ -202,4 +202,13 @@ main.bd-content #main-content dl.simple dt + dd {
 li.nav-item.toc-entry {
   line-height: 1.25em;
   margin-bottom: 0.25em;
-}
\ No newline at end of file
+}
+
+.guilabel, .menuselection {
```

Comment:
span.guilabel, span.menuselection {

to overwrite default settings of pydata, the base theme of books theme.

Reply:
just a small aestetic suggestion: no borders

```
border: none;
padding: 3px 6px 1px 6px;
```

Reply:
I fiddled around with it a little more:

Adjust style for guilabel and menuselection
- Remove border
- Adjust padding to avoid letters that go below the line (j, p, q, etc.) from touching the edge of the padding
- Increase size for legibility, reducing jaggies from anti-aliasing

Reply:
Please let me know what you think, or feel free to adjust it further.